src/Tribe/PUE/Checker.php

```diff
@@ -866,8 +866,11 @@ public function check_for_updates( $updates = array() ) {
 
 			//Is there an update to insert?
 			if ( version_compare( $state->update->version, $this->get_installed_version(), '>' ) ) {
-				$updates['response'] = empty( $updates['response'] ) ? array() : $updates['response'];
-				$updates['response'][ $this->get_plugin_file() ] = $state->update->to_wp_format();
+				if ( empty( $updates ) ) {
+					$updates = array( 'response' => array() );
+					$updates = (object) $updates;
```

Comment:
You _could_ collapse that down to just this:

```php
$updates = (object) array( 'response' => array() );
```

But I wouldn't hate you if you chose not to.